diff --git a/crypto/CMakeLists.txt b/crypto/CMakeLists.txt
index 5140a7a..ca95669 100644
--- a/crypto/CMakeLists.txt
+++ b/crypto/CMakeLists.txt
@@ -841,11 +841,10 @@ add_library(crypto $<TARGET_OBJECTS:crypto_obj> $<TARGET_OBJECTS:compat_obj> emp
 export_symbol(crypto ${CMAKE_CURRENT_BINARY_DIR}/crypto_p.sym)
 target_link_libraries(crypto ${PLATFORM_LIBS})
 set_target_properties(crypto PROPERTIES
-	OUTPUT_NAME crypto
-	ARCHIVE_OUTPUT_NAME crypto
+	OUTPUT_NAME libcrypto
+	ARCHIVE_OUTPUT_NAME libcrypto
 	EXPORT_NAME Crypto
 	VERSION ${CRYPTO_VERSION}
-	SOVERSION ${CRYPTO_MAJOR_VERSION}
 )
 if(NOT CMAKE_VERSION VERSION_LESS 3.27.0)
 	set_target_properties(crypto PROPERTIES DLL_NAME_WITH_SOVERSION TRUE)
