diff --git a/CMakeLists.txt b/CMakeLists.txt
index cc0e904..ed26b53 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -362,7 +362,7 @@ if(CMAKE_SYSTEM_PROCESSOR MATCHES "(aarch64|arm64|ARM64)")
 	if(WIN32)
 		set(ENABLE_ASM false)
 	endif()
-elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "arm")
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(arm|ARM)")
 	set(HOST_ARM true)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "SunOS" AND CMAKE_SYSTEM_PROCESSOR MATCHES "i386")
 	set(HOST_X86_64 true)
